azure_functions_worker/testutils.py

```diff
@@ -239,6 +247,115 @@ def _run_test(self, test, *args, **kwargs):
                     raise test_exception
 
 
+class SharedMemoryTestCase(unittest.TestCase):
+    """
+    For tests involving shared memory data transfer usage.
+    """
+    def setUp(self):
```

Comment:
defining this in the setUp() causes this to happen - would setUpClass be better? Create shm only once?

![image](https://user-images.githubusercontent.com/1047040/111373286-f1fbff00-8658-11eb-8083-0c2e20452baf.png)

Reply:
As long as we dispose this volume off properly, this should be fine. We can test again after those changes.